content/collections/data/en/visual-labeling.md

```diff
@@ -29,26 +31,26 @@ Visual Labeling is available to Amplitude users with the role **Member** and abo
 
 To use Visual Labeling to create new labeled events, follow these steps:
 
-1. Open Amplitude Data and click *Visual Labeling* in the left rail. The Launch Visual Labeling modal appears.
-2. Enter the URL of the site or application you want to label, and click *Start Labeling*. Amplitude opens your website or app in a new tab, with the visual labeling toolbar at the top of the page.
-3. Click an element you want to label, the visual labeling overlay appears. Click *Navigate* to navigate to a different part of site that you want to label.
-4. In the visual labeling overlay, enter a name and description for your labeled event. Select if tracking should happen when a user clicks the element, or when it changes. Visual labeling uses the `clicked` event by default. Refine the definition and select filters as needed. When you're done, click *Save*. If you have labeled events that have the same definition, a warning appears to help you avoid duplication.
+1. Open Amplitude Data and click _Visual Labeling_ in the left rail. The Launch Visual Labeling modal appears.
+2. Enter the URL of the site or application you want to label, and click _Start Labeling_. Amplitude opens your website or app in a new tab, with the visual labeling toolbar at the top of the page.
+3. Click an element you want to label, the visual labeling overlay appears. Click _Navigate_ to navigate to a different part of site that you want to label.
+4. In the visual labeling overlay, enter a name, description, and category for your labeled event. Select if tracking should happen when a user clicks the element, or when it changes. Visual labeling uses the `clicked` event by default. Refine the definition and select filters as needed. When you're done, click _Save_. If you have labeled events that have the same definition, a warning appears to help you avoid duplication.
```

Comment:
@markzegarelli - added a change to the visual labeling page here too. The rest were formatting changes.

Reply:
Thanks @cyhk! LGTM!